doc/can-bind.md

```diff
@@ -49,13 +49,13 @@ The binding instance also has one property, [can-bind.prototype.parentValue],
 which returns the value of the parent observable.
 
   @param {Object} options An object with multiple options:
-    - **child** `{ObservableValue}`: Required; the child observable.
+    - **child** `{ObservableValue|ObservableEmitter}`: Required; the child observable. It should support either [can-symbol/symbols/onValue can.onValue symbol] or [can-symbol/symbols/onEmit can.onEmit symbol] methods.  If [can-symbol/symbols/onEmit can.onEmit symbol] is provided, the parent value will be updated whenever a value is emitted.  If [can-symbol/symbols/onValue can.onValue symbol] is provided, the child value will be updated whenever the value changes.
```

Comment:
Should this say “the child value” instead of “the parent value”?

Reply:
Actually, on second thought, should these say the parent value will be updated, and the `parent` docs should say the child will be updated when the parent emits a value?